packages/apm/src/integrations/tracing.ts

```diff
@@ -402,6 +402,8 @@ export class Tracing implements Integration {
 
     const span = hub.startSpan(
       {
+        op: 'operation',
+        sampled: true,
```

Comment:
FTR (as we talked over video chat), I believe more users will forget to set this and get surprised and frustrated.
It feels like only the SDK itself should ever set the valued of `sampled`, never the user.